klayout.dockerfile

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+
+FROM hdlc/build:build AS build
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    g++ ruby-dev python3-dev qt5-default qt5-qmake qttools5-dev libqt5xmlpatterns5-dev qtmultimedia5-dev libqt5svg5-dev \
```

Comment:
Please, use one line per package. That helps with git, which tracks content by line.

At the same time, clang++ is installed in hdlc/build:build already. Do you know (or did you try) whether klayout can be built with clang?

Reply:
gcc is my default compiler, I leave clang testing to other people.